static/assets/question.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Same here, there's `question-circle` and `question-filled`

Reply:
Same issue as above. AUI component vs figma deisgns (and there doesn't appear to be a way to remove the outer cirlce).

<img width="94" alt="Screen Shot 2022-06-06 at 3 04 52 pm" src="https://user-images.githubusercontent.com/37155488/172098140-c5ad5b0c-5d8e-4a37-82e1-0924f4cb578b.png">